src/dosbox.cpp

```diff
@@ -834,8 +834,19 @@ void DOSBOX_Init()
 	        "   4: 2 MB for the FBI and one TMU with 2 MB (default).\n"
 	        "  12: 4 MB for the FBI and two TMUs, each with 4 MB.");
 
-	pbool = secprop->Add_bool("voodoo_multithreading", only_at_start, true);
-	pbool->Set_help("Use threads to improve 3dfx Voodoo performance (enabled by default).");
+	// Deprecate the boolean Voodoo multithreading setting
+	pbool = secprop->Add_bool("voodoo_multithreading", deprecated, false);
+	pbool->Set_help("Use 'voodoo_threads = yes / no / <value>'.");
+
+	pstring = secprop->Add_string("voodoo_threads", only_at_start, "yes");
+	pstring->Set_help(
+	        "Use threads to improved 3dfx Voodoo performance ('yes' by default).\n"
+	        "  yes:      Use up to 7 threads based on available CPU cores (default).\n"
+	        "  no:       Disable threads: low frame rates with only one core used.\n"
+	        "  <value>:  Set a specific number of threads.\n"
```

Comment:
IMO `auto` is more descriptive than `yes` and `no` can be removed since it's the same as a value of 0.  If you want, you can put a note that these are _additional_ worker threads (the main thread always does Voodoo work as well) so a good value would be core count minus 1.

Reply:
Seconded, should be `auto` and the number option.

Reply:
Great suggestions @weirddan455

Reply:
> I would also consider `none/off` options - just like in `pcspeaker`, `oplmode`, `window_titlebar`, etc.

Will add that @FeralChild64; makes sense, and good to be consistent. Thanks.